src/server/helpers/memcached.js

```diff
@@ -0,0 +1,30 @@
+import Memcached from 'memcached';
+
+import { conf } from '../helpers/config';
+
+let memcached = null;
+
+const getMemcachedStub = () => {
+  return {
+    get: (key, callback) => callback(),
+    set: (key, value, lifetime, callback) => callback()
```

Comment:
I'd suggest making the default stub use a trivial in-memory cache object, the way that some tests do.  That would make it a more accurate stub and would simplify tests further.